app/src/components/v-input.vue

```diff
@@ -182,6 +189,11 @@ function emitValue(event: InputEvent) {
 	if (props.type === 'number') {
 		const parsedNumber = Number(value);
 
+		if (props.integer === true && !Number.isInteger(parsedNumber) && Number.isNaN(parsedNumber) === false) {
```

Comment:
```suggestion
		if (props.integer === true && !Number.isInteger(parsedNumber) && !Number.isNaN(parsedNumber)) {
```

Were you trying to avoid a double negative here?